deployments/charts/kuma/values.yaml

```diff
@@ -56,6 +56,9 @@ controlPlane:
     kubernetes.io/os: linux
     kubernetes.io/arch: amd64
 
+  # -- For affinity configurations
+  affinity: {}
```

Comment:
Should we add `antiAffinity` options as well? That could be important in the High Availability use case as well.

Reply:
Oops, re-reading I realize this is supported as a nested field. Sorry!